### ISSUE TITLE
Cannot find the theme.

### ISSUE THREAD

**Issue by Landixbtw** [cpwrs/americano.nvim#1]
I was trying to use the theme, but it seems that nvim cannot find the theme.
I am using lazy.nvim 
```lua
    {
        enabled = true,
        "cpwrs/americano.nvim",
        config = function()
            vim.cmd.colorscheme("americano")
        end
    },
   ```

**Comment by cpwrs** [comment]
Should be fixed in cb0c3f5! The colorscheme is still work in progress.

**Comment by Landixbtw** [comment]
Thanks, works great. Gonna try it out now.
